lua/autorun/client/cl_pvp_movespeed.lua

```diff
@@ -1,2 +1,8 @@
 -- Includes --
-include( "autorun/shared/sh_pvp_movespeed.lua" )
\ No newline at end of file
+include( "autorun/shared/sh_pvp_movespeed.lua" )
+
+--Sends to server to drop current weapon
+concommand.Add( "cfc_dropweapon", function()
```

Comment:
console commands can be created serverside, so you don't need net message code.

Reply:
Created in the client so it shows up when you're typing it in the console. If it was created server side it wouldn't show the command when typing it.